fundingmanager.go

```diff
@@ -1992,34 +1992,32 @@ func (f *fundingManager) handleFundingConfirmation(
 	fundingPoint := completeChan.FundingOutpoint
 	chanID := lnwire.NewChanIDFromOutPoint(&fundingPoint)
 
-	// Now that the channel has been fully confirmed, we'll mark it as open
```

Comment:
In the commit message:  regitster -> register

Reply:
Fixed.